apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnForeignKey.tsx

```diff
@@ -47,7 +47,7 @@ const ColumnForeignKey = ({
     return {
       id: c.id,
       name: c.name,
-      format: column.format || c.format,
+      format: c.format || column.format,
```

Comment:
Fixes this bug 
<img width="1021" height="686" alt="Screenshot 2025-11-07 at 3 28 19 PM" src="https://github.com/user-attachments/assets/0c8af378-b100-401e-9485-9f7165e55efd" />